src/domain/runtime/event/mod.rs

```diff
@@ -1,27 +1,43 @@
 //![Runtime events](https://chromedevtools.github.io/devtools-protocol/tot/Runtime)
 
-use std::fmt;
-
+use crate::domain::runtime::r#type::RemoteObject;
 use serde::{Deserialize, Serialize};
-
+use std::fmt;
 pub mod console_api_called;
-pub mod exception_thrown;
 
 #[derive(Debug, Serialize, Deserialize)]
-#[serde(tag = "method", content = "params")]
 #[non_exhaustive]
+#[serde(tag = "method", content= "params")]
 pub enum Event {
     #[serde(rename = "Runtime.consoleAPICalled")]
     ConsoleAPICalled(console_api_called::Event),
     #[serde(rename = "Runtime.exceptionThrown")]
-    ExceptionThrown(exception_thrown::Event),
+    ExceptionThrown(ExceptionParams),
+}
+
+#[derive(Debug, Serialize, Deserialize)]
+#[non_exhaustive]
+#[serde(rename_all = "camelCase")]
+pub struct ExceptionParams{
+    pub timestamp: i32,
+    pub exception_details: ExceptionDetails
+}
+
+#[derive(Debug, Serialize, Deserialize)]
+#[serde(rename_all = "camelCase")]
+pub struct ExceptionDetails {
+    pub text: String,
+    pub line_number: i32,
+    pub column_number: i32,
+    pub url: Option<String>,
+    pub exception: RemoteObject,
 }
 
 impl fmt::Display for Event {
     fn fmt(&self, f: &mut fmt::Formatter) -> fmt::Result {
         match &self {
             Event::ConsoleAPICalled(event) => write!(f, "{}", event),
-            Event::ExceptionThrown(event) => write!(f, "{}", event),
+            Event::ExceptionThrown(params) => write!(f, "{}", params.exception_details.text),
```

Comment:
nit: I'd probably prefer `params.exception_details.text.fmt(f)` for better handling of pad/align/etc (and less fmt bloat if you ever `cargo expand` this code but it's super not important and totally reasonable either way (and the other branch already does this so)

Reply:
makes sense, thanks for catching!